coa-helm/templates/csi/blobfuse-proxy.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-blobfuse-proxy
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-blobfuse-proxy
+  template:
+    metadata:
+      labels:
+        app: csi-blobfuse-proxy
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: type
+                    operator: NotIn
+                    values:
+                      - virtual-kubelet
+      initContainers:
+        - name: prepare-binaries
+          image: mcr.microsoft.com/k8s/csi/blob-csi:v1.8.0
+          command: ['sh', '-c', "cp /blobfuse-proxy/*.deb /tmp/"]
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp-dir
+      containers:
+        - command:
+            - nsenter
+            - '--target'
+            - '1'
+            - '--mount'
+            - '--uts'
+            - '--ipc'
+            - '--net'
+            - '--pid'
+            - '--'
+            - sh
+            - '-c'
+            - |
+              set -xe
+              INSTALL_BLOBFUSE=${INSTALL_BLOBFUSE:-true}
```

Comment:
I see that "blobfuse" is some kind of Azure file system driver/enabler/thing. Am I right to guess that this chart basically serves the purpose of setting up Cromwell's Azure file system? Is there more to it, at a high level?